test/extensions/access_loggers/grpc/tcp_grpc_access_log_integration_test.cc

```diff
@@ -53,10 +53,11 @@ class TcpGrpcAccessLogIntegrationTest : public Grpc::GrpcClientIntegrationParamT
     config_helper_.addConfigModifier([this](envoy::config::bootstrap::v2::Bootstrap& bootstrap) {
       auto* listener = bootstrap.mutable_static_resources()->mutable_listeners(0);
       auto* filter_chain = listener->mutable_filter_chains(0);
-      auto* config_blob = filter_chain->mutable_filters(0)->mutable_config();
+      auto* config_blob = filter_chain->mutable_filters(0)->mutable_typed_config();
 
+      ASSERT_TRUE(config_blob->Is<envoy::config::filter::network::tcp_proxy::v2::TcpProxy>());
       envoy::config::filter::network::tcp_proxy::v2::TcpProxy tcp_proxy_config;
-      TestUtility::jsonConvert(*config_blob, tcp_proxy_config);
+      config_blob->UnpackTo(&tcp_proxy_config);
```

Comment:
You probably want to use https://github.com/envoyproxy/envoy/blob/master/source/common/protobuf/utility.h#L290 instead of `UnpackTo`, since this verifies the result.

Reply:
Done